### DISCUSSION TITLE
OpenVPN Tasker plugin disappeared from Google Play Store

### DISCUSSION THREAD

**Discussion by yff77** [schwabe/ics-openvpn#1730] [General]
OpenVPN Tasker plugin disappeared from Google Play Store, is there any official link to download the apk-file? I've found mirrors with the apk, but I'm not sure if I can trust those sites...
I've managed to work with intents in tasker, but only in the foreground (while the tasker plugin works in the background).

**Comment by schwabe** [comment]
That tool was maintained by someone else. I have no more information than you have, sorry.
